test_scripts/clientconf.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"flag"
+	"fmt"
+	"github.com/golang/protobuf/proto"
+	pb "github.com/sergeyfrolov/gotapdance/protobuf"
+	"io/ioutil"
+	"log"
+	"net"
+)
+
+func printClientConf(clientConf pb.ClientConf) {
+	fmt.Printf("Generation: %d\n", clientConf.GetGeneration())
+	fmt.Printf("Default Pubkey: %s\n", hex.EncodeToString(clientConf.GetDefaultPubkey().Key[:]))
+	decoys := clientConf.DecoyList.TlsDecoys
+	fmt.Printf("Decoy List: %d decoys\n", len(decoys))
+	for i, decoy := range decoys {
+		ip := make(net.IP, 4)
+		binary.BigEndian.PutUint32(ip, decoy.GetIpv4Addr())
+		fmt.Printf("%d:\n  %s (%s)\n", i, decoy.GetHostname(), ip.To4().String())
+		if decoy.GetPubkey() != nil {
+			fmt.Printf("  pubkey: %s\n", hex.EncodeToString(decoy.GetPubkey().Key[:]))
+		}
+		if decoy.GetTimeout() != 0 {
+			fmt.Printf("  timeout: %d ms\n", decoy.GetTimeout())
+		}
+		if decoy.GetTcpwin() != 0 {
+			fmt.Printf("  tcpwin: %d bytes\n", decoy.GetTcpwin())
+		}
+	}
+
+}
+
+func parseClientConf(fname string) pb.ClientConf {
+
```

Comment:
Please run `gofmt -w -s .`

Reply:
didn't change anything (my vim configuration already does that on save I think)

Reply:
Hmm, that's weird. I just rechecked and indeed `gofmt` has no issue with extraneous newline. Oh well